docs.md

```diff
@@ -0,0 +1,48 @@
+---
+title: Docs
+---
+<ul class="nav nav-tabs" style="margin-bottom: 20px;">
+    <!-- <li><a href="#">Overview</a></li> -->
```

Comment:
Could these be built from a data object rather than statically defined in the HTML?